experimental/uefi/app/src/main.rs

```diff
@@ -24,7 +24,7 @@
 #![test_runner(crate::test_runner)]
 #![reexport_test_harness_main = "test_main"]
 
-use runtime::echo;
+use runtime::framing;
```

Comment:
I will have to admit that I'd much rather see the (a) renaming of `echo` to `framing` and (b) refactoring the error handling in a separate PR, but it's not like Github makes that easy.